tests/h/util/test_logging_filters.py

```diff
@@ -0,0 +1,69 @@
+# -*- coding: utf-8 -*-
+"""Test logging filters."""
```

Comment:
The "Test logging filters" comment here and below can be removed, since it is obvious that this is a test and what is being tested from the name of the module.